packages/office-ui-fabric-react/src/utilities/selection/Selection.ts

```diff
@@ -172,6 +172,10 @@ export class Selection implements ISelection {
   }
 
   public isRangeSelected(fromIndex: number, count: number): boolean {
+    if (count === 0) {
```

Comment:
Why is this change necessary?

Reply:
This prevents an empty group header from appearing "selected". This is the call site: https://github.com/OfficeDev/office-ui-fabric-react/blob/e9306660b0538cc4c30e6d485bbf575a7c78c28f/packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx#L129

Reply:
It looks like `isRangeSelected` is only ever called directly by GroupedListSection, and indirectly by GroupedList via `toggleRangeSelected`. I've done some manual testing of selecting & deselecting groups, so I think this is a safe change.